### DISCUSSION TITLE
Limit parallel writers to a delta table

### DISCUSSION THREAD

**Discussion by t1g0rz** [delta-io/delta-rs#2517] [Q&A]
Is there a way to limit the number of writers to the table? I noticed that if I use a VM with 8 CPU and 64 GB of memory, I'm able to write to the table. However, if I use 16 CPU and 64 GB of memory, I'm getting status code -9.
I'm using Python binding 0.16.4.

**Comment by rtyler** [comment]
There is [some code which was recently added](https://github.com/delta-io/delta-rs/blob/main/crates/core/src/storage/mod.rs#L189) that introduces a limiting environment variable. I believe that was included in some recent 0.17.x releases if you are up for trying it out
